core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/person/edit/PersonEditViewModel.kt

```diff
@@ -356,29 +359,7 @@ class PersonEditViewModel(
     }
 
     private fun validateUsername(username: String): Boolean {
```

Comment:
This function can be deleted and the validate username use case can be called directly. This function serves no purpose.